README.md

```diff
@@ -1,95 +1,66 @@
+The repository is organized as a _monorepo_, where various ICON model components and utilities are developed as independent Python namespace packages in subfolders. An `icon4py` Python package is defined at the root folder just to collect the specific versions of the different components as project dependencies. The component packages can also be installed independently, although since they are not (yet) available from a package repository, they need to be installed from their specific location within this repository. This can be easily done by following the [Installation instructions](#installation-instructions) explained below.
 
 ## License
 
 ICON4Py is licensed under the terms of the BSD-3-Clause.
 
 ## Installation instructions
-### Dependencies
-A minimal installation of ICON4Py needs 
-- Python 3.10
-- boost >= 1.85.0
 
-You can install all packages at once by using the provided `requirements.txt` or `requirements-dev.txt` files in the root of the repository. For example:
-The `-dev.txt` file installs ICON4Py packages and GT4Py in editable mode, such that source changes are immediatly picked up and used in the virtual environment. 
-```bash
-# Clone the repository
-git clone git@github.com:C2SM/icon4py.git
-cd icon4py
-
-# Create a (Python 3.10) virtual environment (usually at `.venv`)
-python3.10 -m venv .venv
+Since this project is still in a highly experimental state, it is not yet available as a regular Python distribution project through PyPI. The expected installation procedure is to clone the [https://github.com/C2SM/icon4py](https://github.com/C2SM/icon4py) repository and install it in a venv using the following development workflow.  
 
-# Activate the virtual environment and make sure that 'wheel' is installed
-source .venv/bin/activate
-pip install --upgrade wheel pip setuptools
+### System dependencies
 
-# Install all the ICON4Py packages and its dependencies
-# External dependencies would be checked out at './_external_src'
-pip install --src _external_src -r requirements-dev.txt
+ICON4Py requires **_Python >= 3.10_** and **_boost >= 1.85.0_**, and uses the `uv` tool to manage the development workflow. `uv` is a versatile tool which bundles together functionality from different applications: it can work as a _fast_ Python package manager (like `pip`), as a dependency version exporter (like `pip-tools`), as a Python application runner (like `pipx`) or as a full project development manager (like `hatch`). 
+`uv` can be installed in different ways (check its [installation instructions](https://docs.astral.sh/uv/getting-started/installation/)), like using the recommended standalone installer:
 
-# Finally, check that everything works
-pytest -v
+```bash
+$ curl -LsSf https://astral.sh/uv/install.sh | sh 
 ```
+although it can also be installed with `pip` / `pipx`
```

Comment:
would delete this unless there is a reason why you would ever need this fallback